packages/headless/src/controllers/recommendation/headless-recommendation.test.ts

```diff
@@ -7,33 +6,32 @@ import {
 } from '../../features/recommendation/recommendation-actions';
 import {recommendationReducer} from '../../features/recommendation/recommendation-slice';
 import {
-  buildMockRecommendationAppEngine,
-  MockRecommendationEngine,
-} from '../../test/mock-engine';
+  buildMockRecommendationEngine,
+  MockedRecommendationEngine,
+} from '../../test/mock-engine-v2';
+import {createMockRecommendationState} from '../../test/mock-recommendation-state';
 import {
   buildRecommendationList,
   RecommendationList,
 } from './headless-recommendation';
 
+jest.mock('../../features/recommendation/recommendation-actions');
+jest.mock('../../features/pagination/pagination-actions');
+
 describe('headless recommendation', () => {
```

Comment:
```suggestion
describe('Recommendation, () => {
```

for the sake of consistency